runtime/runtime.go

```diff
@@ -13,46 +13,50 @@ import (
 type Options struct {
 	ConnectionCacheSize int
 	MetastoreDriver     string
-	MetastoreDSN        string
 	QueryCacheSizeBytes int64
 	AllowHostAccess     bool
 	SafeSourceRefresh   bool
+	// GlobalDrivers are drivers whose handles are shared with all instances
+	GlobalDrivers []*Connector
```

Comment:
Can we call it `SystemConnectors` or just  `Connectors`? (Since it's on `Runtime` it's clear these are not instance-specific)

Reply:
An unrelated note that I'm putting here – as hinted in another comment, we'd need a `bool` on `Connector` to mark whether a runtime-level connector can be accessed by an instance. But since there are no such cases right now, I would propose we don't add that support and just never allow an instance to access these for now.

Reply:
Let's call it `SystemConnectors`. Will be explicit that these are not instance specific.